<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mesero_cancela_platillos</class>
 <widget class="QDialog" name="mesero_cancela_platillos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>379</x>
     <y>460</y>
     <width>248</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnCancelar">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	color: #539E56;
	border-radius: 10px;
	background-color: #E3EFB0;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}</string>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancelarPlatillos">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>35</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">*{
	color: white;
	border-radius: 10px;
	background-color: #46B04A;
	border: none;
	padding: 5px;
}

:hover{
	background-color: white;
	border: 1px solid #46B04A;
	color: #46B04A;
}

:disabled {
	background-color: rgb(133, 133, 133);
	color: white;
}</string>
      </property>
      <property name="text">
       <string>Cancelar platillos</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>79</y>
     <width>601</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>170</y>
     <width>601</width>
     <height>281</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>599</width>
      <height>279</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridListaPlatillos"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="elPatio_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>51</y>
     <width>239</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Selecciona los platillos a cancelar</string>
   </property>
  </widget>
  <widget class="QLabel" name="elPatio_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>262</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #585858
</string>
   </property>
   <property name="text">
    <string>Cancelar Platillo o Comanda</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="elPatio_6">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
</string>
      </property>
      <property name="text">
       <string>Comanda creado por</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lb_nomMesero">
      <property name="font">
       <font>
        <family>Roboto</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(133, 133, 133);
</string>
      </property>
      <property name="text">
       <string>[NOMBRE_MESERO]</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cBox_CancelarComanda">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>140</y>
     <width>211</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(133, 133, 133);
</string>
   </property>
   <property name="text">
    <string>Cancelar toda la Comanda</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>101</y>
     <width>294</width>
     <height>50</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="elPatio_4">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>Número mesa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_numMesa">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>[NÚMERO_MESA]</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="elPatio_5">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>Número comanda</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_numComanda">
        <property name="font">
         <font>
          <family>Roboto</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(133, 133, 133);
</string>
        </property>
        <property name="text">
         <string>[NÚMERO_COMANDA]</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
